tests/test_other.py

```diff
@@ -3607,18 +3607,19 @@ def test_no_exit_runtime_warnings_flush(self):
 ''')
     warning = 'stdio streams had content in them that was not flushed. you should set EXIT_RUNTIME to 1'
 
-    def test(cxx, no_exit, assertions, flush, keepalive):
+    def test(cxx, no_exit, assertions, flush=0, keepalive=0, filesystem=1):
```

Comment:
We could remove the redundant argument below after this.
```diff
--- a/tests/test_other.py
+++ b/tests/test_other.py
@@ -3636,7 +3636,7 @@ int main() {
       for no_exit in [0, 1]:
         for assertions in [0, 1]:
           for flush in [0, 1]:
-            test(cxx, no_exit, assertions, flush, 0)
+            test(cxx, no_exit, assertions, flush)
 
   def test_fs_after_main(self):
     for args in [[], ['-O1']]:
```
(GitHub does not allow me to comment on that line)